src/overlays/actors/ovl_En_Trt2/z_en_trt2.c

```diff
@@ -76,111 +103,833 @@ static DamageTable D_80AD58D8 = {
     /* Powder Keg     */ DMG_ENTRY(1, 0x0),
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80AD5964[] = {
+void func_80AD3380(SkelAnime* skelAnime, ActorAnimationEntryS* animation, s32 arg2) {
+    f32 phi_f0;
+
+    animation += arg2;
```

Comment:
`animation += arg2` wat

Reply:
Yeah this same form is used in a bunch of actors I've done. I can't get it to match using [], or using a temp, or using `animation= &animation[]` or anything either.